connector/mqtt/latency-pub-test-prometheus/README.md

```diff
@@ -0,0 +1,63 @@
+# Latency pub test prometheus
+
+### Latency between publication MQTT and Kafka with Prometheus and Dojot
+
+A [prometheus](https://github.com/prometheus/prometheus) client to measure the latency between publication MQTT and when payload arrives in Apache Kafka for dojot.
```

Comment:
A [prometheus](https://github.com/prometheus/prometheus) client to measure the payloads latencies between an MQTT broker and Apache Kafka for dojot.